crates/net/Cargo.toml

```diff
@@ -99,3 +100,5 @@ eventsource = [
     'web-sys/EventSource',
     'web-sys/MessageEvent',
 ]
+# As of now, only implements `AsyncRead` and `AsyncWrite` on `WebSocket`
+io-util = ["futures-io"]
```

Comment:
I figured it was best to provide this as an optional feature so that `futures-io` is not added in the dependency graph of all dependents